Cesium.IntegrationTests/type_casts.c

```diff
@@ -0,0 +1,17 @@
+typedef unsigned int size_t;
+
+int foo(int a, int b)
+{
+    return a + b;
+}
+
+int main(void)
+{
+    int a = (int) 1.0;
+    size_t b = foo(1, 2); // ((size_t)) a;
+    size_t c = (foo)(1, 2); // ((size_t)) a;
+    size_t d = (size_t) a;
+    size_t e = (size_t) (a + b);
```

Comment:
Can you please add more expressions like this
```
int f = (int)a;
int g = (int)(a + b);
```
Just to check that primitive types are also works.
Whole expression become more complex, so maybe it would be better to just put parts of it in `if`, something like this https://github.com/ForNeVeR/Cesium/blob/940c1cb23d953a5a03ed7629e8ecfa2d10420b44/Cesium.IntegrationTests/array/multidimensional.c#L5-L10

Reply:
I added test cases for (int) cast